x-pack/plugins/upgrade_assistant/public/application/components/overview/fix_issues_step/fix_issues_step.tsx

```diff
@@ -22,10 +23,39 @@ const i18nTexts = {
   }),
 };
 
-export const getFixIssuesStep = ({ nextMajor }: { nextMajor: number }): EuiStepProps => {
+interface Props extends OverviewStepProps {
+  nextMajor: number;
+}
+
+const FixIssuesStep: FunctionComponent<OverviewStepProps> = ({ setIsComplete }) => {
+  const [isEsFixed, setIsEsFixed] = useState(false);
```

Comment:
[nit] It might be helpful to rename this or add a comment, indicating it representative of when there are 0 critical deprecations. It's clear to me now, but when I first saw `setIsFixed` as a prop in `es_stats.tsx` I had to pause a minute to think about what it represented, but maybe it's still too early for me 😅 .

Reply:
👍  Sure, I added a comment. I think it makes sense to keep it abstract like this, so that we don't need to change function and prop names if we ever change the "is fixed" logic. For example, if we decide users need to fix all "warning" issues too.